source/common/grpc/grpc_web_filter.cc

```diff
@@ -130,5 +168,47 @@ Http::FilterTrailersStatus GrpcWebFilter::encodeTrailers(Http::HeaderMap& traile
   return Http::FilterTrailersStatus::Continue;
 }
 
+void GrpcWebFilter::setupStatTracking(const Http::HeaderMap& headers) {
```

Comment:
Done.
However, a better way could be get a super class for grpc-web filter and http11 bridge filter, whenever the target backend is gRPC. Not sure if it's highly desired. Potential it can be reused for REST to gRPC too.